cronjob/templates/cronjob.yaml

```diff
@@ -23,10 +23,12 @@ spec:
           containers:
             -
               name: app
-              image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
-              command: {{- toYaml .Values.command | nindent 14 }}
+              image: {{ .Values.container.image.repository }}:{{ .Values.container.image.tag }}
```

Comment:
do we expect multiple containers in the same job? just keep the existing one if no